src/bin/vip-import-sql.js

```diff
@@ -121,6 +121,12 @@ const gates = async ( app, env, fileName ) => {
 		);
 	}
 
+	if ( ! env?.importStatus ) {
+		await track( 'import_sql_command_error', { error_type: 'empty-import-status' } );
+		exit.withError(
+			'Unable to check for existing import or database operations on the site.'
```

Comment:
How about:
```suggestion
			'Could not determine the import status for this environment.'
```

Reply:
Not a blocker, but I find it's helpful to include a "what to do next" style suggestion for errors like this to avoid leaving users in a confused state. Should they run it again? Should they check the environment ID? Should they contact support?

Reply:
> How about:

Looks good. I was trying to stay somehat consistent with the messaging for the other two related errors:
https://github.com/Automattic/vip/blob/1ced866721c9e63b5553e34eb0149239c41fad71/src/bin/vip-import-sql.js#L143

This is better overall though.

>Not a blocker, but I find it's helpful to include a "what to do next" style suggestion for errors like this to avoid leaving users in a confused state. Should they run it again? Should they check the environment ID? Should they contact support?
Good point, I was trying to think of something here, but those are good suggestions, especially around checking the siteID.